sdk/keyvault/keyvault-keys/src/cryptographyClient.ts

```diff
@@ -785,32 +785,32 @@ export type KeySignatureAlgorithm =
 /**
  * Options for the encrypt call to the CryptographyClient
  */
-export interface EncryptOptions extends RequestOptions {}
+export interface EncryptOptions extends CryptographyOptions {}
 
 /**
  * Options for the decrypt call to the CryptographyClient
  */
-export interface DecryptOptions extends RequestOptions {}
+export interface DecryptOptions extends CryptographyOptions {}
 
 /**
  * Options for the sign call to the CryptographyClient
  */
-export interface SignOptions extends RequestOptions {}
+export interface SignOptions extends CryptographyOptions {}
 
 /**
  * Options for the verify call to the CryptographyClient
  */
-export interface VerifyOptions extends RequestOptions {}
+export interface VerifyOptions extends CryptographyOptions {}
 
 /**
  * Options for the wrapKey call to the CryptographyClient
  */
-export interface WrapOptions extends RequestOptions {}
+export interface WrapOptions extends CryptographyOptions {}
```

Comment:
The method name is wrapKey does that mean this interface should be WrapKeyOptions?

Reply:
+1 WrapKeyOptions/UnwrapKeyOptions